Postman Test Suite/Record Ownership Test/Record Ownership Manual Setup Scripts.sql

```diff
@@ -0,0 +1,82 @@
+  --Client 1 should be able to full CRUD their School 1.
+  --Derived Resource -Client 1 has full CRUD - School 1
+  --Client 2 should be able to full CRUD School 2.
+  --Derived Resource -Client 2 has full CRUD - School 2
+  --Assuming Client 2 is assigned the extra ownership token, it should also be able to full CRUD School 1. 
+  --But Client 1, without the assignment to Ownership Token 2, should not be able to CRUD School 2.
+
+use EdFi_Admin_Test
+GO
+
+INSERT INTO [dbo].[OwnershipTokens] ([Description])     VALUES  ('OwnershipToken_One')
+INSERT INTO [dbo].[OwnershipTokens] ([Description])     VALUES  ('OwnershipToken_Two')
+
+DECLARE @ApiClient_ApiClientId INT;
+DECLARE @OwnershipToken_OwnershipTokenId INT;
+SELECT @ApiClient_ApiClientId = ApiClientId FROM [dbo].[ApiClients] where Name in ('255901001')
```

Comment:
Are we using the "No Further Authorization Required" strategy for educationOrganizations in the security metadata for this testing, and are we granting full CRUD permissions to educationOrganizations to both of these clients? I ask that because if so, it's a bit irrelevant (even non-intuitive or misleading) if they are API clients affiliated with specific Schools, because typically that would provide relationship-based constraints on what data they are allowed modify.

Ideally, we'd have 2 clients that associated with the same LEA so that there's no concerns over general access to the target resource based on relationships. Then, the only factor that obviously needs to be considered in their 403 when trying to update or delete the resource is the Record Ownership strategy.

Set up the test scenario so that the reason for the 403 is both _distinct_ and _obvious_.

Reply:
2 clients that associated with the same LEA now